pythoncalculator/divide.py

```diff
@@ -0,0 +1,2 @@
+def divide(x, z):
```

Comment:
Hi! There's an undefined 'y' in this function - z! Once this is changed the code should work!